liquidator/liquidator.js

```diff
@@ -412,11 +417,9 @@ class Liquidator {
       let resolvedPrice;
       if (requestTimestamp) {
         try {
-          resolvedPrice = revertWrapper(
-            await this.votingContract.getPrice(this.empIdentifier, requestTimestamp, {
-              from: this.empContract.options.address
-            })
-          );
+          resolvedPrice = await this.votingContract.getPrice(this.empIdentifier, requestTimestamp, {
```

Comment:
Why did you remove the `revertWrapper`? Sometimes `getPrice` will return a really weird number instead of throwing an error which is why we catch that

Reply:
ye, this is a good point. I've added it back accordingly as well as a comment with reference to the OZ thread on why this is needed.